src/tests/simulation/src/slt.rs

```diff
@@ -59,16 +73,54 @@ impl SqlCmd {
 }
 
 fn extract_sql_command(sql: &str) -> SqlCmd {
-    let cmd = sql
-        .trim_start()
-        .split_once(' ')
-        .unwrap_or_default()
-        .0
-        .to_lowercase();
-    match cmd.as_str() {
-        "create" => SqlCmd::Create {
-            is_create_table_as: is_create_table_as(sql),
-        },
+    let sql = sql.to_lowercase();
+    let tokens = sql.split_whitespace();
+    let mut tokens = tokens.multipeek();
+    let first_token = tokens.next().unwrap_or("");
```

Comment:
https://github.com/risingwavelabs/risingwave/pull/13539#discussion_r1400289215

> If we have some tests intended to fail with parse error, will this lead to panic of the test driver?

Removed the `unwraps`.